java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1610,9 +1613,7 @@ public void xadd(BaseClient client) {
                         .xadd(
                                 key,
                                 Map.of(field1, "foo3", field2, "bar3"),
-                                StreamAddOptions.builder()
-                                        .trim(new StreamAddOptions.MaxLen(Boolean.TRUE, 2L))
-                                        .build())
+                                StreamAddOptions.builder().trim(new MaxLen(Boolean.TRUE, 2L)).build())
```

Comment:
This should work too
```suggestion
                                StreamAddOptions.builder().trim(new MaxLen(true, 2L)).build())
```